modelbaker/dbconnector/db_connector.py

```diff
@@ -319,7 +319,13 @@ def get_tid_handling(self):
         Returns `True` if a tid handling is enabled according to the settings table (when the database has been created with `--createTidCol`).
         If t_ili_tids are used only because of a stable id definition in the model (with `OID as` in the topic or the class definition), this parameter is not set and this function will return `False`.
         """
-        return False
+        return {}
```

Comment:
This is probably never called in the base class, so just wondering why changing it?

Reply:
To have the same return type like the derived functions.

Reply:
But this function is returning only True or False in the implementations :thinking: